### ISSUE TITLE
Incompatible with jest ^26.6.3

### ISSUE THREAD

**Issue by OR13** [digitalbazaar/edv-client#81]
```
const edv = require('edv-client');

it('can require edv', () => {
  console.log(edv);
});
```

This will eventually error with `FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory`

when run in jest... I think this is caused by minimal-cipher... 

```
    "jest": "^26.6.3"
```

**Comment by OR13** [comment]
Downgrading jest to

```
  "jest": "22.0.0"
```
resolved this issue.

**Comment by OR13** [comment]
I using tsdx downgrade to :

```
"tsdx": "^0.13.2",
```

**Comment by dlongley** [comment]
Should be addressed by #103.
